<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>p2pWidget</class>
 <widget class="QWidget" name="p2pWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>233</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>P2PWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="controllBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QLineEdit" name="z_3lineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="z_2lineEdit"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="z_4lineEdit"/>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="axisZCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="refVelSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="sliderPosition">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="refVelSpinBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>deg/s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/big_starticon.png</normaloff>
            <normalon>:/icons/big_stopicon.png</normalon>:/icons/big_starticon.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="z_5lineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="x_2lineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="x_4lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="z_1lineEdit"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="x_5lineEdit"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="axisZLabel">
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zPosSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="zPosSpinBox">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="axisXCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="axisXLabel">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="xPosSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="xPosSpinBox">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="x_1lineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="x_3lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusBox">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="axisXLabel_2">
          <property name="text">
           <string>AxisX:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="axisZLabel_2">
          <property name="text">
           <string>AxisZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_2">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_2">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>xPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xPosSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xPosSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xPosSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zPosSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zPosSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zPosSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refVelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refVelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refVelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refVelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
